types/d3-geo/index.d.ts

```diff
@@ -1,6 +1,6 @@
-// Type definitions for D3JS d3-geo module 1.9
+// Type definitions for D3JS d3-geo module 1.10
```

Comment:
Whoops, careful! The d3-geo module is still only at [1.9](https://github.com/d3/d3-geo/releases). This version number tracks the upstream module, not the type definitions. Updates are pushed out as patch bumps.

Reply:
gotcha. i tried 1.9.1 but travis wouldn't allow patch versioning. thought it was appropriate to update the version number to indicate a change.

Reply:
reverted. doesn't matter if we merge this one or the existing PR below. but this is breaking our build. whatever is faster would be preferred. :)